modules/apps/portal-workflow/portal-workflow-kaleo-runtime-api/src/main/java/com/liferay/portal/workflow/kaleo/runtime/constants/WorkflowTasksConstants.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ * <p>
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ * <p>
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.workflow.kaleo.runtime.constants;
+
+/**
+ * @author Sarai Díaz
+ */
+public class WorkflowTasksConstants {
+
+	public static final int WORKFLOW_TASKS_ANY = 0;
```

Comment:
Why? I mean, why do you need the constant and with the normal value for APROVED. I think -1 is ANY.